src/module/item/base/document.ts

```diff
@@ -862,6 +862,40 @@ class ItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Item
             this.actor.update(actorUpdates);
         }
     }
+
+    /** To be overridden by subclasses to extend the HTML string that will become part of the embed */
+    _embedHTMLString(_config: DocumentHTMLEmbedConfig, _options: EnrichmentOptions): string {
+        // prerequisites (PrerequisiteTagData[])
+        const prereq: { value: string }[] | null = foundry.utils.getProperty(this, "system.prerequisites.value");
```

Comment:
Retrieve exactly what you want (and know what you're retrieving) instead of using `getProperty`.